models/br_mg_belohorizonte_smfa_iptu/br_mg_belohorizonte_smfa_iptu__dicionario.sql

```diff
@@ -0,0 +1,10 @@
+{{ config(alias='dicionario', schema='br_mg_belohorizonte_smfa_iptu') }}
+
+
+SELECT 
+SAFE_CAST(id_tabela AS STRING) id_tabela,
+SAFE_CAST(nome_coluna AS STRING) nome_coluna,
+SAFE_CAST(chave AS STRING) chave,
+SAFE_CAST(cobertura_temporal AS STRING) cobertura_temporal,
+REPLACE(REPLACE(valor, '(Zona de Especial Interesse Social - 2', 'Zona de Especial Interesse Social - 2'), '(Zona de Preservação Ambiental', 'Zona de Preservação Ambiental') AS valor
+FROM basedosdados-staging.br_mg_belohorizonte_smfa_iptu_staging.dicionario AS t
```

Comment:
retira o " ( " no valor

Reply:
amiga, eu realmente preciso tirar esse "(" dos dados.

Reply:
é só adicionar mais um replace no modelo dbt msm, acho que pode ser uma boa, tipo na hora que a pessoa for puxar do dicionário o valor daquela chave, vir o nome bonitinho!